build.gradle

```diff
@@ -336,6 +335,7 @@ configure(project(':core')) {
             exclude(module: 'jackson-annotations')
             exclude(module: 'jackson-databind')
             exclude(module: 'bcprov-jdk15on')
+            exclude(group: 'org.slf4j', module: 'slf4j-simple')
```

Comment:
this is not related to the verification, it was just causing a warning in console when running